Extension/CHANGELOG.md

```diff
@@ -1,61 +1,115 @@
 # C/C++ for Visual Studio Code Changelog
 
-## Version 1.22.8: October 7, 2024
+## Version 1.22.8: October 8, 2024
+### Bug Fixes
+* Fix an issue with lingering incorrect squiggles after an edit. [#12175](https://github.com/microsoft/vscode-cpptools/issues/12175)
+* Reverted/portponed changes related to recursive includes handling, due to issues with some projects.
```

Comment:
handling, due to issues with some projects->handling due to issues with some projects

Comment should be removed.